tools/fluent/basic_syntax.md

```diff
@@ -104,11 +104,39 @@ emails = { $unreadEmails ->
 
 Note how the `[one]` form in English doesn’t explicitly use the variable, in order to display the word `one` instead of the digit `1`.
 
-In plural messages is always possible to expose the number, even if the reference language doesn’t; the name of the variable is defined at the beginning of the select expression, in this case `$unreadEmails`. The first example in this section can be translated in Italian using `$unreadEmails` in both forms:
+In plural messages is always possible to expose the number, even if the reference language doesn’t; the name of the variable is defined at the beginning of the select expression (in the *selector*), in this case `$unreadEmails`. The first example in this section can be translated in Italian using `$unreadEmails` in both forms:
 
 ```PROPERTIES
 emails = { $unreadEmails ->
         [one] C’è { $unreadEmails } messaggio non letto.
        *[other] Ci sono { $unreadEmails } messaggi non letti.
     }
 ```
+
+## Variants and Terms
+
+As described at the beginning of the document, **terms** are a special type of **messages**: they can be recognized because of the identifier starting with a dash, e.g. `-brand-short-name`, and can only be referenced within other messages. More important, they are allowed to define additional attributes compared to the reference language (typically `en-US`).
+
+While in most cases terms will have a single value, it’s also possible to define multiple **variants**: different values, each one associated to a key, that can be selected when referencing the term ID from another message (or term). Unlike select expressions with a selector (e.g. the `emails` message in the previous section), the value retrieved doesn’t depend on external variables or functions.
+
+This can be useful, for example, to adapt a message to different grammatical cases or usages. Consider the following example in English:
+
+```PROPERTIES
+-fxaccount-brand-name = Firefox Account
+sync-signedout-account-title = Connect with a { -fxaccount-brand-name }
+```
+
+In Italian this can become:
+
+```PROPERTIES
+-fxaccount-brand-name =
+    {
+        [lowercase] account Firefox
+       *[uppercase] Account Firefox
+    }
+sync-signedout-account-title = Connetti il tuo { -fxaccount-brand-name[lowercase] }
+```
+
+Similar to select expressions, you must define a default variant, identified by `*`. Also notice that key names are arbitrary, and don’t need to be in English.
+
+It’s important to note that this entire section applies to terms, not messages: while it’s possible to define variants in messages using a select expressions without a selector, such variants cannot be retrieved from other messages.
```

Comment:
I'd remove this last sentence. Again, this is an obscure part of the design. I think the rest of the section makes it clear that variants only make sense when defined on terms.